src/sentry/feedback/usecases/create_feedback.py

```diff
@@ -98,6 +100,11 @@ def create_feedback_issue(event, project_id):
     # make sure event data is valid for issue platform
     validate_issue_platform_event_schema(event_data)
 
+    project = Project.objects.get_from_cache(id=project_id)
```

Comment:
@JoshFerge is it possible that loading this from cache could cause us to send multiple "first" events?

Reply:
maybe -- we do the same thing in replays and it seems to work okay.